sandbox/biosdk-service-mz.properties

```diff
@@ -0,0 +1,14 @@
+#iam mock-sdk config
+sdk_check_iso_timestamp_format=true
+mosip.role.biosdk.getservicestatus=REGISTRATION_PROCESSOR
+biosdk_bioapi_impl=io.mosip.mock.sdk.impl.SampleSDKV2
+
+mosip.biosdk.log-request-response-enabled=true
+
+logging.level.root=INFO
+logging.level.io.mosip.biosdk.services.impl.spec_1_0.BioSdkServiceProviderImpl_V_1_0=DEBUG
+
+#iam image-compressor config
+mosip.bio.image.compressor.resize.factor.fx=0.25
+mosip.bio.image.compressor.resize.factor.fy=0.25
+mosip.bio.image.compressor.compression.ratio=50
```

Comment:
we need a PR of this first in 1.2.0.1-v3 branch of config to be in sync with release-1.2.0.1 branch of the apps

Reply:
Created PR: https://github.com/mosip/mosip-config/pull/4675